musicbrainz/PutJSONTogether/releasesAPI.js

```diff
@@ -0,0 +1,34 @@
+var fs = require("fs");
```

Comment:
Is this script still needed? It is not getting called from /musicbrainz/musicbrainzcli/server.js